plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -55,6 +56,26 @@ export default function MySubscriptions(): JSX.Element {
 			! subscriptionsInstalled.includes( subscription )
 	);
 
+	if ( ! wccomSettings?.isConnected ) {
+		return (
+			<div className="woocommerce-marketplace__my-subscriptions--connect">
+				<div className="woocommerce-marketplace__my-subscriptions__icon" />
+				<h2 className="woocommerce-marketplace__my-subscriptions__header">
+					{ __( 'Manage your subscriptions', 'woocommerce' ) }
+				</h2>
+				<p className="woocommerce-marketplace__my-subscriptions__table-description">
```

Comment:
I find `table-description` a bit out of place here, since there is no table. Did you want to re-use the rules? Is it possible to rename this to something else?

Reply:
Good catch, it was left over from copy pasting. I changed it now.